engine/apps/webhooks/tasks/trigger_webhook.py

```diff
@@ -172,7 +189,12 @@ def make_request(webhook, alert_group, data):
 
 
 @shared_dedicated_queue_retry_task(
-    autoretry_for=(Exception,), retry_backoff=True, max_retries=1 if settings.DEBUG else 3
+    autoretry_for=(Exception,),
+    # This allows to exclude some exceptions that match autoretry_for but for which you don’t want a retry.
+    # https://docs.celeryq.dev/en/stable/userguide/tasks.html#Task.dont_autoretry_for
+    dont_autoretry_for=(requests.exceptions.Timeout,),
+    retry_backoff=True,
+    max_retries=1 if settings.DEBUG else 3,
```

Comment:
main change (rest is just adding some type hints)

Reply:
An alternative here would be to _not_ use `dont_autoretry_for` and instead manually retry the task if `isinstance(exception, requests.exceptions.Timeout)` (up to a max of 3 after which point we simply `return`). This would allow to still retry for these tasks but without having to raise an exception (and avoiding triggering our `AmixrRetriedFailedTasks` alert)

Reply:
I think this would be preferred to have some limited retry to cover customer network hiccups